website/registrations/forms.py

```diff
@@ -99,6 +99,8 @@ class RenewalForm(forms.ModelForm):
         required=False, label=_("I am an employee of iCIS")
     )
 
+    contribution = forms.IntegerField(required=False,)
```

Comment:
Now the field is never required, but it is for benefactors.

Reply:
I tested this, and apparently the `clean()` will still require this